command/pipelines/pipeline_execute.go

```diff
@@ -0,0 +1,99 @@
+package pipelines
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/spinnaker/spin/command"
+	gate "github.com/spinnaker/spin/gateapi"
+)
+
+type PipelineExecuteCommand struct {
+	ApiMeta command.ApiMeta
+
+	application string
+	name        string
+}
+
+// flagSet adds all options for this command to the flagset and returns the
+// flagset object for further modification by subcommands.
+func (c *PipelineExecuteCommand) flagSet() *flag.FlagSet {
+	cmd := "pipeline execute"
+
+	f := c.ApiMeta.GlobalFlagSet(cmd)
+	f.StringVar(&c.application, "application", "", "Spinnaker application the pipeline lives in")
```

Comment:
Are parameters & artifacts being handled in a separate PR?

Reply:
Yes, I want to get the rudimentary operations done before expanding.